Framework/Crystal/test/FindSXPeaksTest.h

```diff
@@ -34,6 +35,15 @@ void makeOnePeak(size_t histo, double peak_intensity, size_t at_bin,
   overWriteSpectraY(histo, workspace, peaksInY);
 }
 
+// Helper method to round a double value to ten decimal place
+double roundTo10Places(double toRound) {
+  double scaleFactor = 10e10;
+  double result = toRound * scaleFactor;
+  result = round(result);
+  result = result / scaleFactor;
+  return result;
+}
+
```

Comment:
If you are doing this only to accommodate floating point errors in the tests, then I would suggest to use [this](http://cxxtest.com/guide.html#ts_assert_delta)